dashboard/app/controllers/registrations_controller.rb

```diff
@@ -32,10 +32,10 @@ def update
             redirect_to after_update_path_for(@user)
           end
         end
-        format.json { head :no_content }
+        format.any { head :no_content }
```

Comment:
Repro steps:

Age interstitial:
1. Log in as a teacher
2. Go to http://localhost.code.org:3000/teacher-dashboard/#/sections
  - create a section with login type picture
  - create a student without specifying age
  - click show secret to see their secret picture
3. Open the /section url for that section in a new incognito window
  - log in as a student
  - see age interstitial
  - select age and click continue
Expected: interstitial goes away
Current behavior on prod: nothing appears to happen when clicking Continue

Edit account:
1. Sign in as any account
2. Go to "Edit Account" in user menu: http://localhost-studio.code.org:3000/users/edit
3. Change Display Name
4. Click [Update]
Expected (and current behavior on prod): Go back to edit account page with a success message at the top in green

Edit account with error:
1. Sign in as any account
2. Go to "Edit Account" in user menu: http://localhost-studio.code.org:3000/users/edit
3. Change Email to "a@a" (something that passes the js regex to enable the submit button but isn't actually a valid email)
4. Click [Update]
Expected (and current behavior on prod): Go back to edit account page with error messages in red and invalid fields highlighted